3.preprocess-features/3b.normalize-features/3b.normalize-merged-features.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### Import Libraries
+# 
+
+# In[1]:
+
+
+import pathlib
+import joblib
+import importlib
+
+import pandas as pd
+import numpy as np
+from sklearn.preprocessing import StandardScaler
+from pycytominer.cyto_utils import DeepProfiler_processing
+
+normalization_utils = importlib.import_module("normalization-utils")
+
+
+# ### Set Load/Save Paths
+# 
+
+# In[2]:
+
+
+# paths to load merged features,index, and annotations from
+merged_features_save_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-merged/"
+)
+dp_index_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-DP/inputs/metadata/index.csv"
+)
+annotations_path = pathlib.Path(
+    "../0.image-download/manifest/idr0080-screenA-annotation.csv"
+)
+
+# path to save normalized merged features to
+normalized_merged_features_save_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-merged-normalized/"
+)
+normalized_merged_features_save_path.mkdir(parents=True, exist_ok=True)
+scaler_save_dir = pathlib.Path("normalization-scalers/")
+scaler_save_dir.mkdir(parents=True, exist_ok=True)
+
+
+# ### Normalize merged single-cell data
+# 
+
+# In[3]:
+
+
```

Comment:
consider providing a single comment providing a general explanations of what is occurring in this for loop.